Allow setting custom chrome options

dudeofawesome opened the pull request:
Headless Chrome in Travis CI only works with sandboxing disabled, so I needed to a way to set that option with this module. 

I added a `--chrome-option` argument which can be used multiple times to create an array of options that get added to the `commandLineOptions` array.

I've currently got a working version of this published on NPM at `@dudeofawesome/chrome-headless-render-pdf`.

Szpadel commented on cli/chrome-headless-render-pdf.js:
```diff
@@ -58,6 +58,13 @@ if (typeof argv['chrome-binary'] === 'string') {
 }
 
+let chromeOptions = [];
+for (let i = 0; i < process.argv.length; i++) {
```
Please use minimist for handling arguments

Szpadel replied:
IMHO `--chrome-option --no-sandbox` looks like both options are for parent command. And may be misleading. I guess just better indication. How to correctly pass additional arguments in help text is better solution.

dudeofawesome replied:
Fair point. I'd been quoting my params which made it more readable, but I think you're right.

Szpadel commented on index.js:
```diff
@@ -186,10 +187,10 @@ class RenderPDF {
+             `--remote-debugging-port=${this.port}`,
              '--disable-gpu'
-            ];
+            ].concat(this.options.chromeOptions);
```
Please change it to use array unpack from es6
```
[ 'a', 'b', 'c', ...options ];
```

dudeofawesome commented:
@Szpadel I believe I've addressed all of your points now. 

I'm not super thrilled with the description I've got for how to use the new `chrome-option` flag, so if you've got something better, I'll change it for sure.
